pkgs/applications/blockchains/bitcoin-classic.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchFromGitHub, pkg-config, autoreconfHook, openssl, db48, boost
-, zlib, miniupnpc, qtbase ? null, qttools ? null, util-linux, protobuf, qrencode, libevent
```

Comment:
Some of the bitcoin packages will break because one variant uses qt5's callPackage and one does not.

Reply:
See the first PR commit which takes care of these issues. Note how none of the bitcoin packages are being rebuilt :)